general/src/store/normalize.rs

```diff
@@ -65,6 +65,10 @@ pub fn is_valid_platform(platform: &str) -> bool {
     VALID_PLATFORMS.contains(&platform)
 }
 
+pub fn is_valid_environment(environment: &str) -> bool {
```

Comment:
Wondering if this function should be in parity with this one: https://github.com/getsentry/sentry/blob/3fd6386943efb8a5737e0e471ba85998321c18f1/src/sentry/models/environment.py#L45-L54

this was introduced in https://github.com/getsentry/sentry/pull/9815

Reply:
@dashed good point. This part is handled here:
https://github.com/getsentry/semaphore/blob/b95b6a2dec2bf611d610b66f6df2543756b5be49/general/src/protocol/event.rs#L285-L287

Reply:
Gotcha thanks 👍